src/content/docs/mobile-apps/dashboards-new-relic-mobile-apps.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Dashboards in New Relic mobile apps
+tags:
+  - Mobile apps
+  - New Relic mobile apps
+  - Authentication and alerts
+metaDescription: 'In New Relic mobile you can use our dashboard capability to gather and chart the specific data you want to see, the way you want to see it, from anywhere in our platform.'
+---
+import dashboardsDashboardstoPrepareforMobile from 'images/dashboards_screenshot-full_dashboards-to-prepare-for-mobile.png'
+
+import dashboardsMobileDashboards from 'images/dashboards_screenshot-full_mobile-dashboards.png'
+
+import dashboardsFilterMobileDashboardsbyVariable from 'images/dashboards_screenshot-full_filter-mobile-dashboards-by-variable.png'
+
+
+In New Relic Mobile, you can use our dashboard capability to gather and chart the specific data you want to see, the way you want to see it, from anywhere in our platform.
+
+Any dashboards created in the New Relic website will be visible in the mobile app. In the mobile app you can quickly read and react, having the data you need in your pocket. To create new dashboards, please log into New Relic on the web.
+
+* Drive insight with custom, high-density interactive visualizations with a consistent UI.
+* Chart all the events and attributes from everywhere across our platform. 
+* Add custom attributes or send custom event types to most events in order to better understand your business, and see specific details about how your customers interact with your platform, such as page views, host transactions, etc.
+* Explore and contextualize data with advanced tooltips and zoom functions to monitor what your systems are doing in real time.
+* Search your dashboards for attributes and metrics.
+* Report custom data to New Relic and your dashboards from any source.
+
+## Getting started with Dashboards [#getting-started]
+
+1. Log into your New Relic Mobile app and tap “Dashboards”
+2. For each dashboard, the index displays the following information:
+    * Favorite status, indicated by a star
+    * Name: The name of the dashboard
+    * Created by: The user who created the dashboard
+    * Last edited: When the dashboard was last modified
+    * Created on: When the dashboard was created
+
+## Dashboard permissions
+
+Dashboards have three types of permissions:
+Edit - everyone in account: All users have full rights to the dashboard.
+Read-only - everyone in account: All users are able to see the dashboard, but only you've full rights to work with the dashboard. Other users can access the dashboard but are not able to edit or delete it, although they can duplicate it.
+Private: Only you can see the dashboard. Everything but the metadata is hidden.
+
+To edit permissions, please visit the New Relic website.
```

Comment:
I'd add here a link. Or the one for the NR website or the one where we explain how to edit permissions.